macOS/DuckDuckGo/Onboarding/ContextualOnboarding/ContextualDialogsManager.swift

```diff
@@ -101,7 +101,7 @@ public class ContextualDialogsManager: ObservableObject, ContextualOnboardingDia
     private weak var lastTab: Tab?
 
     // Publisher for contextual onboarding completion
-    @Published private(set) var isContextualOnboardingCompleted: Bool = true
+    @Published private(set) var isContextualOnboardingCompleted: Bool
```

Comment:
Changed to set this value up in the init below, otherwise we might kick off the timer early by publishing a `true` value.